common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -74,6 +74,6 @@
     <div class="capa_alert">${submitted_message}</div>
     %endif
     % if msg:
-    <span class="message">${HTML(msg)}</span>
+    <span class="message" aria-describedby="answer_${id}-legend" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
The value here for aria-describedby is not correct. I found this by using the Axe Chrome plugin and doing an accessibility audit of the page.

![image](https://cloud.githubusercontent.com/assets/484484/18177063/65e96444-7046-11e6-92c8-facba157f055.png)